asabiris/orchestration/render.py

```diff
@@ -17,14 +17,15 @@ def __init__(self, app):
 		self.JinjaService = app.get_service("JinjaService")
 		self.HtmlToPdfService = app.get_service("HtmlToPdfService")
 		self.MarkdownToHTMLService = app.get_service("MarkdownToHTMLService")
-
+		self.TempPath = "/Template/render"
```

Comment:
- [ ] `/Templates` should be configurable (`asab.config`) and used everywhere.
- [ ] `/Templates/Generic`
- [ ] `/Templates` (not `/Template`)

Reply:
@ateska Sure , I will